TypeScript/publish.ts

```diff
@@ -55,99 +48,81 @@ export function publish(
 				options as unknown as ValidPublishOptions;
 
 			function c(e: unknown) {
-				console.error("publish command didnt work for", module.name!);
+				console.error("publish command didnt work for", module.name);
 				console.error(e);
 			}
-			try {
-				const commandData = {
-					type: CommandTypeRaw[module.type],
-					name: module.name!,
-					description: [CommandType.Slash, CommandType.Both].includes(
-						module.type
-					)
-						? module.description
-						: undefined,
-					options: [CommandType.Slash, CommandType.Both].includes(
-						module.type
-					)
-						? optionsTransformer(
-								(module as SlashCommand).options ?? []
-						  )
-						: [],
+			const log = (...message : any[]) => () => console.log(...message);
```

Comment:
so lazy?

Reply:
kinda, its just to put it into callbacks